examples/immutable-values/immutable_values.bal

```diff
@@ -21,37 +21,37 @@ type Employee object {
 };
 
 public function main() {
-    // Create an `anydata` typed `map` with two entries.
+    // Creates an `anydata`-typed `map` with two entries.
     map<string|int> m1 = { stringVal: "str", intVal: 1 };
 
-    // Freeze the map `m1` and assign the returned value to another variable.
+    // Freezes the map `m1` and assigns the returned value to another variable.
     // Since the map is of type `anydata`, the return value would be of the same type as `m1`.
     map<string|int> m2 = m1.freeze();
 
     // Reference equality checks for `m1` and `m2` should evaluate to true, since the same value is returned with the
     // frozen flag set.
     io:println("m1 === m2: ", m1 === m2);
 
-    // Check if `m1` is frozen.
+    // Checks if `m1` is frozen.
     io:println("Frozen status of m1: ", m1.isFrozen());
 
-    // Attempt adding an entry to the `map`, and `trap` the panic if it results in a panic.
+    // Attempts to add an entry to the `map`, and `trap` the panic if it results in a panic.
     error? updateResult = trap addEntryToMap(m2, "intValTwo", 10);
     if (updateResult is error) {
-        // An error should occur since `m2` is frozen
+        // An error should occur since `m2` is frozen.
         io:println("Error occurred on update: ",
                    <string>updateResult.detail().message);
     }
 
-    // Create a `Department` record.
+    // Creates a `Department` record.
     Department d = { name: "finance", id: 1100 };
 
-    // Create a `map` that may hold `anydata|error` typed values.
+    // Creates a `map` that may hold `anydata|error`-typed values.
     // (`map<anydata|error>` belongs to `anydata`)
     map<any> m3 = { stringVal: "str", intVal: 1, recVal: d };
 
-    // Attempt freezing `m3`. Note how the return type could now be an `error`, since there is the possibility that a
-    // `map` constrained by type `any` could have non-anydata values.
+    // Attempts to freeze `m3`. The return type could now be an `error` since there is the possibility that a
```

Comment:
Why are we removing "Note how the" ? We kinda want to emphasize the change.

Reply:
Normally, we don't use words like "note or "please" in technical documents.